docs/scenarios_authoring/BasicNodes.md

```diff
@@ -108,7 +113,10 @@ then nodes that follow `for-each` will be executed twice and the value of curren
 
 ![union_window](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_union0.png)
 
-Union merges multiple branches into one branch. Events from the incoming branches are passed to the output branch without an attempt to combine or match them.
+Union merges multiple branches into one branch. 
+
+In the Streaming processing mode events from the incoming branches are passed to the output branch without an attempt to combine or match them.
+In the Request - Response mode the Union node merges parallel branches of processing back to one branch. You will need to take an extra step using the [Collect](./RRDataSourcesAndSinks.md#collect) node if you need to collect results of processing in each of the merged branches. 
```

Comment:
It suggests that the Union works differently in RR. But AFAIU this is not true. The semantics of RR scenario processing is different. The semantics requires adding the Collect node to "convert X results comes from X Split's arms/pipes into one result"

Reply:
We rewrote this fragment together